internal/shared/types/app_manifest.go

```diff
@@ -252,6 +253,13 @@ type ManifestSlashCommandsItem struct {
 	UsageHint    string `json:"usage_hint,omitempty" yaml:"usage_hint,omitempty"`
 }
 
+// Search contains the search function of an app.
+type Search struct {
+	SearchFunctionCallbackID        string `json:"search_function_callback_id,omitempty" yaml:"search_function_callback_id,omitempty"`
+	SearchFiltersFunctionCallbackID string `json:"search_filters_function_callback_id,omitempty" yaml:"search_filters_function_callback_id,omitempty"`
+	EnableAIAnswers                 *bool  `json:"enable_ai_answers,omitempty" yaml:"enable_ai_answers,omitempty"`
```

Comment:
thought: Don't love the `*bool` pointer but we have used this in other manifest properties, so I think it's acceptable. Is there a default value that the Slack API uses for this field? If not, totally understand that the bool pointer is the right choice. It's just a little dangerous to access because it can lead to runtime segfaults.

Reply:
@mwbrooks I felt a similar hesitation but found a `bool` value would default to `false` and caused the included JSON `omitempty` attribute to not take effect.

Super agree that this can cause runtime issues later. I wonder if a linter can catch accesses to `nil` values? 👾